mcstatus/scripts/address_tools.py

```diff
@@ -0,0 +1,15 @@
+import socket
+from urllib.parse import urlparse
+from ipaddress import ip_address
+
+def ip_type(address):
+    try:
+        return ip_address(address).version
```

Comment:
Aww man, I had a nice long comment here that did not attach itself for some reason. I'm not going to retype all of it. The gist was that people will likely use DNS names for any of their IPv6 addresses (since those are hard to memorize). This code would silently pass None when a ValueError is raised by `ip_address("some.ipv6.host.com")`.

Reply:
ip_type is only used in UDPSocketConnection, so ip_type only analyse the DNS result.

Reply:
You're right. Turns out that dns is already resolved before passing to the connection classes. My bad.